### PR TITLE
Add capitals distances 

### PR DESCRIPTION
**Pull request by bespilotnayakartoshka**

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -10,6 +10,13 @@ use Mix.Config
 config :co2_offset,
   ecto_repos: [Co2Offset.Repo]
 
+# Configurates database
+config :co2_offset, Co2Offset.Repo,
```
Comment (markevich) [inline config/config.exs]: you have `dev.secret.exs` file for this)

File: priv/repo/migrations/20190211103356_create_capitals_distances.exs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+defmodule Co2Offset.Repo.Migrations.CreateCapitalsDistances do
+  use Ecto.Migration
+
+  def change do
+    create table(:capitals_distances) do
+      add :capital_a, :string
+      add :capital_b, :string
+      add :distance, :float
```
Comment (markevich) [inline priv/repo/migrations/20190211103356_create_capitals_distances.exs]: Please add index for distance.
Primary use of the table will be finding the rows with value closes to distance column

Comment (bespilotnayakartoshka) [reply priv/repo/migrations/20190211103356_create_capitals_distances.exs]: added

**Comment by markevich** [comment]
<3

**Pull request merged**